### DISCUSSION TITLE
Dupes

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#1997] [Ideas / Feature Requests]
Perhaps it would be an advantage to be able to check for and eliminate dupes within CloudLog. I know there is a check for dupes on IMPORT, which is great. But if some squeak through, it woujld be nice to be able to check and  correct them from within.

**Comment by AndreasK79** [comment]
Click the search button on top and go to duplicate QSOs.

**Reply by R1BLH** [reply]
Sometimes a duplicate search will show different QSOs as duplicates (with the same date, band and modulation, but close times). 
![2023-02-10_12-00-15](https://user-images.githubusercontent.com/16667173/218048629-c4a41cfc-0413-45e0-8c51-1002189ef225.png)
![2023-02-10_11-59-49](https://user-images.githubusercontent.com/16667173/218048635-7282f54d-bf5b-41a3-afdf-afe47f4345e2.png)

**Reply by AndreasK79** [reply]
Yes, that is correct. It is by design.
